### ISSUE TITLE
[REQ] improve experience when editing bottom of long note

### ISSUE THREAD

**Issue by gingerbeardman** [glushchenko/fsnotes#288]
When editing a long note, more than a screen in length, the cursor is almost always stuck at the bottom of the window and makes editing quite unpleasant.

It would be nice to be able to not have the bottom of the document tied to the bottom of the window.

A large bottom margin would suffice. 
Or maybe there's an even better solution?

![screen shot 2018-07-18 at 21 29 06](https://user-images.githubusercontent.com/49612/42906038-bfd671b6-8ad1-11e8-9638-eab0a67c17ed.png)

**Comment by peavine** [comment]
I would also find it useful to have a blank margin at the bottom of a note. Not anything too large--perhaps the equivalent of one line of text with the line-spacing preference option set at its smallest setting.

**Comment by gingerbeardman** [comment]
Yes, when I said "large" I didn't mean half a page or anything :) 

A line might suffice. Of course, I could put that in manually but I'd rather it be automatic

**Comment by gingerbeardman** [comment]
Thanks!

**Comment by gingerbeardman** [comment]
I can't help but feel there's a better way to do this?

Scroll View > Content Inset > Bottom = 10
rather than
Constraints > Bottom Space = 10

<img width="248" alt="screen shot 2018-08-15 at 15 31 00" src="https://user-images.githubusercontent.com/49612/44153475-42a09a52-a0a0-11e8-8c21-efcdadb00fbb.png">

That way the scrollbar would go to the bottom and the text would not be cut off like this.

<img width="637" alt="screen shot 2018-08-15 at 15 13 01" src="https://user-images.githubusercontent.com/49612/44152547-bdb808e0-a09d-11e8-8d2c-e27f997eabee.png">

**Comment by glushchenko** [comment]
> Scroll View > Content Inset > Bottom = 10

Does not work for me. Provide please working example.

**Comment by gingerbeardman** [comment]
OK, I'll do this

**Comment by gingerbeardman** [comment]
[The Archive](url
https://github.com/glushchenko/fsnotes/issues/217#issuecomment-430616282) calls this (not locking the bottom of the document to the bottom of the window) "Typewriter Mode"

**Comment by gingerbeardman** [comment]
I'd still love to see this.

**Comment by dkmar** [comment]
+1 for some sort of overscrolling. I found an article [here](https://christiantietze.de/posts/2017/07/typewriter-mode-overscrolling/) about implementing it.

**Comment by gingerbeardman** [comment]
Would still love to see this.

I hate myself every time I add dozens of blank lines at the end of a note.

Any chance @glushchenko ?

**Comment by glushchenko** [comment]
https://github.com/glushchenko/fsnotes/releases/tag/7.0.0b1
